docs/README.md

```diff
@@ -1,47 +1,196 @@
-check specific metadata (like check name and description). Users can configure custom checks the same way built-in checks
-are configured, and add them to the config file. The built-in checks are specified [here](../internal/builtinchecks).
-
-### Ignoring violations for specific cases
-
-To ignore violations for specific objects, users can add an annotation with the key
-`ignore-check.kube-linter.io/<check-name>`. We strongly encourage adding an explanation as the value for the annotation.
-For example, to ignore a check named "privileged" for a specific deployment, you can add an annotation like:
-`ignore-check.kube-linter.io/privileged: "This deployment needs to run as privileged because it needs kernel access"`.
-
-To ignore _all_ checks for a specific object, you can use the special annotation key `kube-linter.io/ignore-all`.
+# KubeLinter
+
+KubeLinter analyzes Kubernetes YAML files and Helm charts and checks them
+against various best practices, with a focus on production readiness and
+security.
+
+KubeLinter runs sensible default checks designed to give you useful information
+about your Kubernetes YAML files and Helm charts. Use it to check early and
+often for security misconfigurations and DevOps best practices. Some common
+issues that KubeLinter identifies are running containers as a non-root user,
+enforcing least privilege, and storing sensitive information only in secrets.
+
+KubeLinter is configurable, so you can enable and disable checks and create your
+custom checks, depending on the policies you want to follow within your
+organization. When a lint check fails, KubeLinter also reports recommendations
+for resolving any potential issues and returns a non-zero exit code.
+
+> [!WARNING]
+> KubeLinter is at an early stage of development. There may be breaking changes
+> in the future to the command usage, flags, and configuration file formats.
+> However, we encourage you to use KubeLinter to test your environment YAML
+> files, see what breaks, and
+> [contribute](https://github.com/stackrox/kube-linter/blob/main/CONTRIBUTING.md)
+> to its development.
+
+## Installing KubeLinter
+
+### Using Go
+
+To install by using [Go](https://golang.org/), run the following command:
+
+```bash
+GO111MODULE=on go get golang.stackrox.io/kube-linter/cmd/kube-linter
+```
+Otherwise, download the latest binary from
+[Releases](https://github.com/stackrox/kube-linter/releases) and add it to your
+PATH.
+
+### Using Homebrew
+
+To install by using [Homebrew](https://brew.sh/) on macOS, Linux, and [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about),
+run the following command:
+
+```bash
+brew install kube-linter
+```
+
+### Using Docker
+
+1. Get the latest KubeLinter Docker image:
+   ```bash
+   docker pull stackrox/kube-linter
+   ```
+   > [!NOTE] We recommend that you alway use the latest image unless you are
```

Comment:
Sorry for not being clear, I meant the opposite. We should encourage people to use an image corresponding to a specific release from https://github.com/stackrox/kube-linter/releases (ideally the latest one) and _not_ use the `latest` tag since that may end up breaking their builds unexpectedly.